modules/mono/editor/GodotTools/GodotTools/Build/BuildSystem.cs

```diff
@@ -139,8 +139,8 @@ public static int Publish(BuildInfo buildInfo, Action<string> stdOutHandler, Act
             // `dotnet clean` / `dotnet build` commands
             arguments.Add(buildInfo.OnlyClean ? "clean" : "build");
 
-            // Solution
-            arguments.Add(buildInfo.Solution);
```

Comment:
Was it necessary to apply the same change to normal builds? I don't feel confident this will not introduce any regression for some users. If this doesn't solve anything, I prefer to keep the old one, and limit the change to publishing only.

Reply:
I don't think it was necessary but I thought we'd want to be consistent. I can open a PR to revert it, but wouldn't it be better to leave it for at least one RC to check if users complain?

Reply:
I checked, and it's also needed for the `dotnet build` command.

Reply:
> I checked, and it's also needed for the dotnet build command.

What is it needed for? We don't use `--output` in normal builds.

Reply:
Oh sorry, I misunderstood. I meant to say that the `dotnet build` command does not work with `--output`, just like `dotnet publish`. In our case, since we already don't use `--output` with `dotnet build`, you are right that we don't need it.

I still fail to see why building the solution would be better. We have released RC2 with these changes, so if you are worried about regressions I guess we can wait and see if users report any issues. But if you still want me to revert this line anyway, I can open a PR.

Reply:
My concern is that there could be multi-projects solutions where some of the projects are not referenced by the main one, but they're still meant to be built. They may run some tasks, or their isolated output could be desired.

But I guess we can always change this back if we receive complaints.